packages/gatsby-source-contentful/src/gatsby-node.js

```diff
@@ -72,6 +72,9 @@ export const pluginOptionsSchema = ({ Joi }) =>
 You can pass in any other options available in the [contentful.js SDK](https://github.com/contentful/contentful.js#configuration).`
         )
         .default(false),
+      disableImageCdn: Joi.boolean()
```

Comment:
What are your thoughts on flipping this logic to this so that this more truly an opt in feature?
```suggestion
      enableImageCdn: Joi.boolean()
```